IstexDoc2Vec.py

```diff
@@ -72,7 +73,33 @@ def LoadDocumentsIstexAndUCBL(self, directory, ucbl) :
 					self.index["ISTEX"+doc["doi"]] = str(self.count)
 					self.count +=1
 				except : pass
-		print self.count
+## Wikipedia data itteration
+		for sub in os.listdir(wiki):
+			subdir = wiki + sub
+			for fname in os.listdir(subdir):
+				if self.wiki_count > 12000000:
+					break
+				for line in open(os.path.join(subdir, fname)):
+					if len(line.split()) < 2 or line[:8] == '<doc id=':
+						continue
+					else:
+						try:
```

Comment:
From line 86 to 98, I would prefer to make this part of the code as a separated function.